android/libraries/rib-android/src/main/java/com/uber/rib/core/RibView.kt

```diff
@@ -0,0 +1,8 @@
+package com.uber.rib.core
+
+import android.view.ViewGroup
+
+interface RibView {
```

Comment:
Do we still need this interface, as long as we use views as the basis?

Reply:
But exactly, we don't use Android views as the basis, but just interfaces that usually extend consumer / observable source. This should also make testing easier. Do you see a better way of doing it?

Reply:
Just realized that it actually gives some room for view implementation, let's leave it as it is